HTAN.model.jsonld

```diff
@@ -9293,10 +9392,103 @@
                     "@id": "bts:HTANParentDataFileID"
                 },
                 {
-                    "@id": "bts:AssayType"
+                    "@id": "bts:MSAssayCategory"
                 },
                 {
-                    "@id": "bts:BatchCorrectionMethod"
+                    "@id": "bts:SoftwareandVersion"
+                },
+                {
+                    "@id": "bts:MassSpectrometryAuxiliaryFile"
+                }
+            ],
+            "sms:validationRules": []
+        },
+        {
+            "@id": "bts:MassSpectrometryLevel4",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "Mass Spectrometry derived data that includes proteomics, metabolomics, and lipidomics, level 4",
+            "rdfs:label": "MassSpectrometryLevel4",
+            "rdfs:subClassOf": [
+                {
+                    "@id": "bts:Assay"
+                }
+            ],
+            "schema:isPartOf": {
+                "@id": "http://schema.biothings.io"
+            },
+            "sms:displayName": "Mass Spectrometry Level 4",
+            "sms:required": "sms:false",
+            "sms:requiresComponent": [
+                {
+                    "@id": "bts:MassSpectrometryLevel3"
+                }
+            ],
+            "sms:requiresDependency": [
+                {
+                    "@id": "bts:Component"
+                },
+                {
+                    "@id": "bts:filename"
+                },
+                {
+                    "@id": "bts:fileFormat"
```

Comment:
Note that `File Format` in CSV is getting changed to `fileFormat` in JSON-LD. We have an open issue on this. No action required here.